ebs_snapshots/snapshot_manager.py

```diff
@@ -119,7 +186,87 @@ def _ensure_snapshot(connection, volume, interval, name):
     elif interval == 'yearly' and min_delta > 3600*24*365:
         _create_snapshot(connection, volume, name)
     else:
-        logging.info(kayvee.formatLog("ebs-snapshots", "info", "no snapshot needed", {"volume": volume.id}))
+        logging.info(kayvee.formatLog("ebs-snapshots", "info", "no snapshot needed", {"volume": volume.id, "lastest_snapshot_id": latest_snapshot_id}))
+
+    # Make a backup copy of latest completed snapshot if there is one completed.
+    backup_id = None
+    if latest_complete_snapshot_id is None:
```

Comment:
Nit. I think these if-statements (and the ones above) could be simplified if an `internalToSeconds` object was used.  Ex:

```python
internalToSeconds = {
	'hourly': 3600,
	'daily': 3600*24,
	'weekly': 3600*24*7,
	'monthly': 3600*24*30,
	'yearly': 3600*24*365,
}

if latest_complete_snapshot_id is None:
    logging.info(...)
elif internalToSeconds.get(interval, default_value) <= min_complete_snapshot_delta:
    backup_id = _copy_snapshot(backup_client, volume, latest_complete_snapshot_id, name)
else:
	logging.warning(...)
```